app/models/saved_claim/burial.rb

```diff
@@ -1,11 +1,14 @@
 # frozen_string_literal: true
 class SavedClaim::Burial < SavedClaim
   FORM = '21P-530'
+  CONFIRMATION = 'PEN'
+  PERSISTENT_CLASS = PersistentAttachment::PensionBurial
+  ATTACHMENT_KEYS = [:transportationReceipts, :deathCertificate].freeze
```

Comment:
IMO these sorts of things are a bit cleaner when defined as instance methods that are not implemented on the base class and then return the appropriate values on the inheriting models, but it's another thing that could go either way so it seems fine. It's just that all of the `self.class::ATTACHMENT_KEYS` stuff it introduces into the code feels a little janky and non-idiomatic.